README.md

```diff
@@ -1,6 +1,12 @@
 # Support
 
-Forms that create tickets for requests coming from government agencies or from GOV.UK contact forms.
+This app:
+
+- Presents anonymous feedback about pages on GOV.UK in a "feedback explorer". Anonymous feedback is collected via the [feedback app](https://github.com/alphagov/feedback) and retrieved from the Support API.
+
+- Hosts internal forms for publishers to create Zendesk tickets, as well as emergency contact info. Emergency contacts are secret and retrieved from an [environment variable](https://github.com/alphagov/govuk-puppet/blob/941373ab48ae50a9d1929caee73a52390004bf81/modules/govuk/manifests/apps/support.pp#L129) at runtime.
+
+- Exposes [internal APIs](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/support.rb) to create Zendesk tickets e.g. via the [feedback app]([feedback app](https://github.com/alphagov/feedback).
```

Comment:
Rather than repeating the link to the Feedback app, I think it would be more useful to link to the line where it creates the Zendesk ticket:

```suggestion
- Exposes [internal APIs](https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/support.rb) to create Zendesk tickets e.g. [via the feedback app](https://github.com/alphagov/feedback/blob/7e6893c0e80d9c98f6cea27d1bd089621054b177/app/models/contact_ticket.rb#L35)
```

Reply:
Oops! Fixed with the new link in the third commit 👍.